canonical/ubuntu.com
# Update privacy notice and TOS for Credentials

## Done

- Replace CUBE privacy notice (/legal/data-privacy/cube-beta) with Credentials privacy notice (/legal/data-privacy/credentials)
- Replace CUBE TOS (/legal/terms-and-policies/cube-terms) with Credentials TOS (/legal/terms-and-policies/credentials-terms)


## QA

- Visit https://ubuntu-com-12599.demos.haus/legal/data-privacy/credentials
  - Verify [copy](https://docs.google.com/document/d/1Ovw3ypWDQwxWZ_tU_2U57uWUW-437v6R2UIMX6wfYYg/edit#)
- Visit https://ubuntu-com-12599.demos.haus/legal/terms-and-policies/credentials-terms
  - Verify [copy](https://docs.google.com/document/d/1vRtCO-FDMC8rqthi4sgI1Fn3ofT6zqcCpjHbOyiyhjY/edit#)
- Visit https://ubuntu-com-12599.demos.haus/legal/data-privacy
  - Verify that "Canonical Credentials privacy notice ›" links to https://ubuntu-com-12599.demos.haus/legal/data-privacy/credentials
- Visit https://ubuntu-com-12599.demos.haus/legal/terms-and-policies
  - Verify that "View Canonical Credentials terms ›" links to https://ubuntu-com-12599.demos.haus/legal/terms-and-policies/credentials-terms

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/CRED-258

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
